### PR TITLE
Update requests to include CSRF Token header

### PR DESCRIPTION
**Pull request by 2Cubed**
Comply with API changes.

### CODE REVIEW

**Comment by 2Cubed** [comment]
cc: @ProbablePrime

File: README.md [hunk 1]
```diff
@@ -8,14 +8,10 @@ We've built a chat bot that logs incoming messages to the console, and sends "Hi
...
 
-# copy the config and fill it in with your details
-cp config.example.py config.py
-vi config.py
+## License
 
-# start the example chatbot
-python example.py
-```
+Public domain: MIT License ([text](https://opensource.org/licenses/MIT))
```
Comment (ProbablePrime) [inline README.md]: :fire: this line.

Comment (2Cubed) [inline README.md]: What even... I didn't add that... o.O
Fixing now.

**Comment by ProbablePrime** [comment]
LGTM. we could add 461 retrying but as this library just connects to chat that shouldn't occur.

We'll look at it when we expand the client to immplement more endpoints

**Comment by 2Cubed** [comment]
@ProbablePrime Sounds good. I also plan to rewrite the entire thing, replacing `tornado` with `asyncio` (as per @connor4312's request).

**Comment by 2Cubed** [comment]
`flake8` is salty. ._.
One sec.

**Comment by 2Cubed** [comment]
There we go. Ready for merge. @ProbablePrime

**Comment by connor4312** [comment]
As Richard mentioned retrying is ideal, but the scope of this client is so narrow that it makes no difference here. Lgmt.

**Pull request merged**